api/v1alpha1/dataset_types.go

```diff
@@ -64,7 +64,6 @@ type DataSetStatus struct {
 // +kubebuilder:printcolumn:name="VolumeStatus",type="string",JSONPath=".status.pvcStatus"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
 // +kubebuilder:printcolumn:name="ErrorCount",type="integer",JSONPath=".status.errorCount"
-// +kubebuilder:printcolumn:name="Errors",type="string",JSONPath=".status.errorsString"
```

Comment:
This column is usually a very long JSON string if present, I find that it makes the entire table less readable when errors occur. Since the users can still get this information from the Studio or by using a combination of `kubectl get -o=json` and `jq`, are we going to keep this field in as an additional status column?

Reply:
Is this information available through Studio?

Reply:
Yes, the Studio has a popup window to show it.